src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ConverterDataToObjImpl;
+import core.basesyntax.dao.ReaderDataFromFile;
+import core.basesyntax.dao.ReaderDataFromFileImpl;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.ReportWriterToFileImpl;
+import core.basesyntax.service.TransactionStrategy;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final String filePath = "src/main/resources/data.CSV";
+        final String reportPath = "src/main/resources/report.CSV";
+
+        ReaderDataFromFile readerDataFromFile = new ReaderDataFromFileImpl();
+        ConverterDataToObjImpl converterDataToObj = new ConverterDataToObjImpl();
+        TransactionStrategy transactionStrategy = new TransactionStrategy();
+        ReportWriterToFileImpl reportWriterToFile = new ReportWriterToFileImpl();
+        ReportCreator reportCreator = new ReportCreator();
+        converterDataToObj.convertData(filePath, readerDataFromFile);
```

Comment:
read and convert should be separate actions
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/d6ebd235-9470-4543-9119-5953c473b4f9)